doc/source/admin/jobs.md

```diff
@@ -126,6 +126,51 @@ The images used for containers can either be specified explicitely in the ``<des
 tool requirements of the Galaxy tool being executed. In this latter case the image is specified by the
 tool using a ``<container>`` tag in the ``<requirements>`` section.
 
+### Running jobs on a Kubernetes cluster via Pulsar
+
+In order to dispatch jobs to a Kubernetes (K8s) cluster via Pulsar, 
+Pulsar implements a "two-container" architecture per pod, where 
+one container stages job execution environment (`pulsar-container`)
+and another container encompasses tool's executables (`tool-container`).
+
+Note that this architecture is experimental and under active development, 
+it is increasingly improved and it will soon be production-grade ready.
+
+In order to setup Galaxy to use the "two-container" architecture, you may 
+take the following steps: 
+
+1. In the `galaxy.yml` set the following attributes: 
+
+- `job_config_file: job_conf.yml`
+- `galaxy_infrastructure_url: 'http://host.docker.internal:$UWSGI_PORT'`
```

Comment:
That's specific for GIEs on macOS, you could just say that this should be configured appropriately.

Reply:
Thanks.